### PR TITLE
New parser for "ulimit_soft"

### PR DESCRIPTION
**Pull request by xiangce**

### CODE REVIEW

**Comment by xiangce** [comment]
Hey @jobselko,  The results of `ulimit -a` are needed by a rule of insights-plugins, and before making this PR, I noticed that you have made a `UlimitHard` parser already in the `prodsec-rules`, so I would try the best to make the parsers in this PR to compatible with the one you have made. Please kindly review and check if it works for your rules. Thanks a lot.

**Comment by jobselko** [comment]
Hi @xiangce. After all, I used  `limits_conf` combiner instead of `ulimit_hard` parser. You can rewrite this parser if needed (remove `data` method and so on).

File: insights/parsers/ulimit.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+"""
+Ulimit - Command ``ulimit -a -H`` and ``ulimit -a -S``
+======================================================
+Parses the output of the `ulimit -a -S` and `ulimit -a -S` command:
```
Comment (jobselko) [inline insights/parsers/ulimit.py]: Typo: `ulimit -a -H` and `ulimit -a -S`

Comment (xiangce) [reply insights/parsers/ulimit.py]: Thanks! Fixed.

**Comment by xiangce** [comment]
@jobselko, Thanks for the response!

**Pull request merged**